src/components/TodoList.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { User } from '../types/User';
+import { Todo } from '../types/Todo';
+
+import './TodoList.scss';
+
+type Props = {
+  todos: Todo[];
+  users: User[];
+};
+
+export class TodoList extends React.PureComponent<Props> {
+  phoneNumber = (user: User) => (
+    user.phone.split('').filter(el => !Number.isNaN(+el)).slice(0, 11).join('')
+  );
+
+  visibleNumber = (str: string) => (
+    str.length > 10
+      ? `+${str[0]} (${str.slice(1, 4)}) ${str.slice(4, 7)} ${str.slice(7)}`
+      : `(${str.slice(0, 3)}) ${str.slice(3, 6)} ${str.slice(6)}`
+  );
+
+  render() {
+    const { todos, users } = this.props;
+
+    return (
+      <div className="content__wrapper">
+        <h2 className="title content__title">
+          Todo list
+        </h2>
+        <ul className="todo-list content__todo-list">
+          {todos.map(todo => {
+            const user = users.find(person => person.id === todo.userId);
+            const phone = user ? this.phoneNumber(user) : '';
```

Comment:
instead of searching user for each todo inside the list it'd be better to create a preparedTodos array basing on todos, where ach prepared todo should have a user property with a link to a user from users array or null if we can't find a use by userId. It should be stored in the App.